tests/envs/test_hovercraft_env.py

```diff
@@ -15,17 +20,29 @@ def test_hovercraft_env():
     env = HoverCraftEnv()
 
     # Uncomment to create video.
+    # from pathlib import Path
```

Comment:
it looks like the unit test will always create videos now. that's fine as long as it doesn't take too long, which it probably doesn't in this case. we can go ahead and remove the outdated commented-out code